opendevin/runtime/plugins/agent_skills/agentskills.py

```diff
@@ -264,169 +330,103 @@ def create_file(filename: str) -> None:
     print(f'[File {filename} created.]')
 
 
-@update_pwd_decorator
-def edit_file(file_name: str, start: int, end: int, content: str) -> None:
-    """Edit a file.
-
-    Replaces in given file `file_name` the lines `start` through `end` (inclusive) with the given text `content`.
+def _edit_or_append_file(
+    file_name: str,
+    start: int | None = None,
+    end: int | None = None,
+    content: str = '',
+    isAppend: bool = False,
+) -> None:
+    """Internal method to handle common logic for edit_/append_file methods.
 
     Args:
-        file_name: str: The name of the file to edit.
-        start: int: The start line number. Must satisfy start >= 1.
-        end: int: The end line number. Must satisfy start <= end <= number of lines in the file.
-        content: str: The content to replace the lines with.
+        file_name: str: The name of the file to edit or append to.
+        start: int | None = None: The start line number for editing. Ignored if isAppend is True.
+        end: int | None = None: The end line number for editing. Ignored if isAppend is True.
+        content: str: The content to replace the lines with or to append.
+        isAppend: bool = False: Whether to append content to the file instead of editing.
```

Comment:
Do we want to use snake case `is_append` - just for consistency?

Reply:
Sure, will change it, was my old habit from dayjob. :)